src/pages/VsCodeExtension.vue

```diff
@@ -184,6 +199,51 @@ export default {
       }
     },
 
+    emitSelectedObject(definition) {
+      this.$root.$emit('selectedObject', definition);
+    },
+
+    setSelectedObject(objectDefinition) {
+      const defMatch = objectDefinition.match(/^([a-z]+):(.+)/);
+      const type = defMatch[1];
+      const object = defMatch[2];
+
+      /* eslint-disable no-case-declarations */
+      switch (type) {
```

Comment:
This kind of API would be useful on `AppMap`
```js
appmap.get('label:authentication')
appmap.get('event:19')
appmap.get('object:com/myorg/MyClass#myMethod')
```

Reply:
Done https://github.com/applandinc/appmap-js/pull/128/commits/c6b275659c7937dfb3dd27848b1bdf63b8065590